code-dot-org/code-dot-org
# Standards: Add button for lesson status and create report

PLEASE MAKE ALL SUGGESTIONS AND FEEDBACK ON: https://github.com/code-dot-org/code-dot-org/pull/32444

# Description

Create two buttons to show the LessonStatusDialog and the CreateReportDialog in the Standards View of the Progress Tab of Teacher Dashboard.

![Screen Shot 2019-12-17 at 12 20 15 PM](https://user-images.githubusercontent.com/208083/71019590-a01ca000-20c8-11ea-8916-3f9a3e2fe509.png)

![buttons-standards-page](https://user-images.githubusercontent.com/208083/71025372-e5df6580-20d4-11ea-8468-e7b15823140b.gif)

Also I made a redux store for standards which has fake data in it right now but I figured it got us one step closer to what we will need down the road.

## Links



- [spec](https://docs.google.com/document/d/1xlm_syljGXIapl72GaKxN63V7kt6syHyvWHuqZhTCfE/edit#)
- [jira](https://codedotorg.atlassian.net/browse/LP-1099)

## Testing story



Add unit tests for:
- Changes to make a small size for button
- Check that the new buttons on the standards view show the appropriate dialogs
- Check that the new buttons only show for the standards view and not other views

# Reviewer Checklist:
